README.md

```diff
@@ -27,6 +27,12 @@ CTRL is distributed under the terms of the [MIT licence](LICENSE).
   - [Deploying to Heroku](#deployingtoheroku)
 
 ## <a id="installation"></a> Installation
+
+Starting from release `v1.0.0` CTRL follows the [Gitflow model](https://nvie.com/posts/a-successful-git-branching-model/).
```

Comment:
Just a short explanation to make sure people understand where things are. The README anyway needs some work...